src/kOS.Safe/Encapsulation/RangeValue.cs

```diff
@@ -0,0 +1,117 @@
+﻿using kOS.Safe.Encapsulation;
+using kOS.Safe.Encapsulation.Suffixes;
+using kOS.Safe.Exceptions;
+using kOS.Safe.Serialization;
+using System;
+using System.Collections.Generic;
+
+namespace kOS.Safe
+{
+    [kOS.Safe.Utilities.KOSNomenclature("Range")]
+    public class RangeValue : EnumerableValue<ScalarIntValue, Range>
+    {
+        private const string DumpFrom = "from";
+        private const string DumpTo = "to";
```

Comment:
See comment above in the .rst doc about it being strange to use the words "from" and "to" to describe a range with an open endpoint that doesn't actually go all the way TO the "to" value.